### DISCUSSION TITLE
Obligation de changement de mot de passe de quel droit?????!!!!!!

### DISCUSSION THREAD

**Discussion by Damedesang** [community/community#35332] [Code Security]
Alors desoler je ne parle que Français Aujourd'huis je me connecte sur mon GitHub et la surprise votre site m'oblige oui oui j'ai bien dit OBLIGE  a changer mon mot de passe. De quel droit faitent vous cela? Celon l'explication (en anglais ...) mon mot de passe est utiliser par moi meme sur d'autre site ou application et cela ne vous convient pas ... 
Donc je vais etre tres clair j'ai des soucis de mémoire mon mot de passe est régulièrement le meme donc m'obliger a le changer est juste débile car je risque juste de le perdre a nouveau... Le noté? quel intérêt de mettre un mot de passe si c'est pour le noté a ce compte la autant ne pas en mettre... 
Vous vous rendez compte si on devait avoir un mot de passe different pour chaque truc qu'on crée a notre époque c'est juste délirant il fauderai un dictionnaire complet pour tout gardé en mémoire.... 
Quand a savoir que j'utilise le meme mot de passe sur d'autre chose j'aimerai bien savoir comment vous savez cela si ce n'est en allant trifouiller dans des choses qui ne vous regarde pas , or je ne pense qu'aller surveiller mes mot de passe sur des site ou application qui ne vous appartiennent pas **ce n'est pas tres légal...**
Et de quel droit vous me verrouillé l'acces a mon compte si je n'en change pas... Précisément il s'agit de MON compte ... Bref coup de geule terminer au détracteur qui me parlerons de sécurité nianiania .... Sachez que piraté mon compte GitHub actuellement n'apporterai rien a personne compte tenue que je debut en matière de développement et que je ne travaille que sur des exercices pour mes cours... Donc si c'est pour dire cela merci de passer votre chemin....

**Comment by tzztntt** [comment]
Hello, I'm Marin.

I've read your post carefully and it seems terrible.

As you know, passwords usually consist of numbers, special characters and numbers.

Your password may be insecure, In my opinion, GitHub will guarantee the security of your privacy.

If I were you, I'd use powerful irregular password and keep it deeply :)

If you forget the password, you can reset the password using Gmail.

Please contact with me if you need my help.
